lib/kasket.rb

```diff
@@ -14,7 +15,7 @@ module Kasket
   autoload :SelectManagerMixin,     'kasket/select_manager_mixin'
   autoload :RelationMixin,          'kasket/relation_mixin'
 
-  CONFIGURATION = {:max_collection_size => 100, :write_through => false}
+  CONFIGURATION = { max_collection_size: 100, write_through: false } # rubocop:disable Style/MutableConstant
```

Comment:
in test we modify the constant ... wanted a quick PR to fix all the simple things first